core/runtime/src/main/java/io/quarkus/runtime/logging/LogBuildTimeConfig.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.runtime.logging;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "log", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class LogBuildTimeConfig {
+
+    /**
+     * Whether or not logging metrics are published in case a metrics extension is present.
+     */
+    @ConfigItem(name = "metrics.enabled", defaultValue = "false")
+    public boolean metricsEnabled;
```

Comment:
Why make this optional when the memory usage is negligible and the CPU overhead would be only a small fraction of the cost of logging the message to any handler?  And why make it a build-time property?

IMO it would be simpler and more expedient to leave it on all the time.

Reply:
Btw the activation may not only be about resource cost, but also a way for users to skip metrics they don't care about (since the time series will consume disk space in Prometheus DB).

Reply:
With other extensions offering metrics, we decided to default to false too, so +1 from me to staying consistent